osu.Game/Rulesets/Mods/Mod.cs

```diff
@@ -42,6 +47,48 @@ public abstract class Mod : IMod, IJsonSerializable
         [JsonIgnore]
         public virtual string Description => string.Empty;
 
+        /// <summary>
+        /// The tooltip to display for this mod when used in a <see cref="ModIcon"/>.
+        /// </summary>
+        /// <remarks>
+        /// Differs from <see cref="Name"/>, as the value of attributes (AR, CS, etc) changeable via the mod
+        /// are displayed in the tooltip.
+        /// </remarks>
+        [JsonIgnore]
+        public virtual string IconTooltip
+        {
+            get
+            {
+                List<string> attributes = new List<string>();
+
+                foreach ((SettingSourceAttribute attr, System.Reflection.PropertyInfo property) in this.GetOrderedSettingsSourceProperties())
```

Comment:
Would prefer having a `using` statement for `System.Reflection` instead of this fully qualified name for `PropertyInfo`.